cmd/serve/grant.go

```diff
@@ -102,6 +102,16 @@ func init() {
 		Bind("gemini-api-key").
 		Env("GEMINI_API_KEY")
 
+	flagBuilder.Flag().String("gemini-api-secret", "",
+		"the api secret for gemini").
+		Bind("gemini-api-secret").
+		Env("GEMINI_API_SECRET")
+
+	flagBuilder.Flag().String("gemini-browser-client-id", "",
+		"the client id for gemini").
```

Comment:
Perhaps put in a better description? Ditto with the above because at-a-glance it's not clear what this ClientID does versus the API-Key

Reply:
resolved: https://github.com/brave-intl/bat-go/pull/881/commits/16ce255a704425f8dcec556994fbbee8ca667fb2